article/app/views/liveblog/liveBlogBodyContentAMP.scala.html

```diff
@@ -0,0 +1,61 @@
+
+@import common.Edition
+@import conf.switches.Switches.LiveUpdateAmpSwitch
+@import views.support.{AmpAd, AmpAdDataSlot}
+@import org.joda.time.DateTimeZone
+
+@liveBlogBlocksAMP(article: Article, timezone: DateTimeZone) = {
+    @model.currentPage.currentPage.blocks.sliding(5, 5).zipWithIndex.map { case (blockGroup, index) =>
+        @views.html.liveblog.liveBlogBlocks(blockGroup, article, timezone, amp = true)
+
+        @* Add advert every 5 blog posts, up to a maximum of 8 *@
+        @if(blockGroup.length == 5 && index < 8) {
+            <div @if(LiveUpdateAmpSwitch.isSwitchedOn) { id="amp-ad-@index" data-sort-time="1" } class="block amp-ad-container amp-ad-container--live-blog">
+                <amp-ad width="300" height="250" type="doubleclick"
+                json=@AmpAd(article, request.path, Edition(request).id.toLowerCase()).toString()
+                data-slot=@AmpAdDataSlot(article).toString()>
+                </amp-ad>
+            </div>
+        }
+    }
+}
+
+@defining((model.article, Edition(request).timezone)) { case (article, timezone) =>
+    <div class="js-article__container" data-component="body">
+        @if(model.currentPage.currentPage.isArchivePage) {
+            @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage)
+        }
+        @if(LiveUpdateAmpSwitch.isSwitchedOn) {
+            <amp-live-list id="live-list" data-poll-interval="15000" data-max-items-per-page="20"
+            class="js-liveblog-body u-cf from-content-api js-blog-blocks blocks @if(article.fields.isLive) {live-blog}" data-most-recent-block="block-@{
+                _root_.liveblog.LatestBlock(article.blocks)
+            }" data-test-id="live-blog-blocks"
+            itemprop="@if(article.tags.isReview) {reviewBody} else {articleBody}">
+                <div update on="tap:live-list.update" class="toast__space-reserver">
+                    <div id="toast__tofix" class="toast__container is-sticky">
+                        <button class="toast__button toast__button--closed button button--large">
+                            <span class="toast__text">New updates</span>
+                            @fragments.inlineSvg("refresh", "icon", List(""))
+                        </button>
+                    </div>
+                </div>
+                <div items>
+                    @liveBlogBlocksAMP(article, timezone)
+                </div>
+            </amp-live-list>
+        } else {
+            <div class="js-liveblog-body u-cf from-content-api js-blog-blocks blocks @if(article.fields.isLive) {live-blog}" data-most-recent-block="block-@{
+                _root_.liveblog.LatestBlock(article.blocks)
+            }" data-test-id="live-blog-blocks"
+            itemprop="@if(article.tags.isReview) {reviewBody} else {articleBody}">
+                @liveBlogBlocksAMP(article, timezone)
+            </div>
+        }
+
+        @* Remove LiveFilter and the fragment once Toast comes out of AB test status *@
```

Comment:
@NataliaLKB It is not clear what this comment is referring to. Doing some GitHub digging, it looks like this comment should probably have been removed at the same time as a duplicate of the comment was removed back in February

https://github.com/guardian/frontend/commit/0058b24f1d4c9a7323bccafacd06281ef5a42606

I think that's what I'll do, to prevent this question being asked again in the future!